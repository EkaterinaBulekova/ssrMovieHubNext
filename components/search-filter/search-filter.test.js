import React from 'react';
import { shallow } from 'enzyme';
import SearchFilter from './search-filter';
import toJson from 'enzyme-to-json';

describe('<SearchFilter />', () => {
  it('should render buttons with onClick method', () => {
    const testOnClick = jest.fn();
    const component = shallow(<SearchFilter/>);
    expect(toJson(component)).toMatchSnapshot();

    // const buttons = component.find('button');

    // expect(buttons).toHaveLength(2);
    // buttons.map(button=>{
    //   button.simulate('click');
    // })
    // expect(testOnClick.mock.calls.length).toBe(2);
    // expect(testOnClick).toBeCalled();

    // component.unmount();
  });
});