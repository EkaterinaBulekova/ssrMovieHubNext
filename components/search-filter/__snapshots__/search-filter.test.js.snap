// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchFilter /> should render buttons with onClick method 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
